sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -777,6 +777,18 @@ def _add_argparse_args(cls, parser):
          'job submission, the files will be staged in the staging area '
          '(--staging_location option) and the workers will install them in '
          'same order they were specified on the command line.'))
+    parser.add_argument(
```

Comment:
Shouldn't the environment that contains the Java SDK harness already contain itself and all of the necessary dependencies to execute the cross language IO transform?

Reply:
Currently under discussion within https://lists.apache.org/thread.html/6fcee7047f53cf1c0636fb65367ef70842016d57effe2e5795c4137d@%3Cdev.beam.apache.org%3E